tests/integration_tests/configs/default.jsonnet

```diff
@@ -4,13 +4,10 @@
     cmd: 'ethermintd',
     'start-flags': '--trace',
     config: {
-      consensus: {
-        // larger timeout for more stable mempool tests
-        timeout_commit: '2s',
-      },
       mempool: {
         // use v1 mempool to enable tx prioritization
         version: 'v1',
+        recheck: false,
```

Comment:
This issue was solved on PR https://github.com/evmos/ethermint/pull/1566
```suggestion
```

Reply:
I didnt debug too deep, thanks for the fix!